src/components/organisms/OakHeader/OakHeader.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { ReactElement, ReactNode } from "react";
+import styled, { css } from "styled-components";
+
+import {
+  OakFlex,
+  OakP,
+  OakHeading,
+  OakBox,
+  OakImage,
+  OakThemeProvider,
+  OakMaxWidth,
+} from "@/components/atoms";
+import { oakDefaultTheme } from "@/styles";
+
+export type OakHeaderProps = {
+  headingTitle: string;
+  heroImageSrc: string;
+  authorImageSrc: string;
+  authorName: string;
+  authorTitle: string;
+  subHeadingText: string;
+  breadcrumbs?: ReactElement;
+  children?: ReactNode;
+};
+
+const StyledAutorImage = styled(OakImage)`
+  width: 50px;
+  height: 50px;
+  margin-right: 12px;
+  img {
+    border-radius: 50%;
+  }
+`;
+
+const OakHeaderCss = css<OakHeaderProps>``;
+
+const UnstyledComponent = (props: OakHeaderProps) => {
+  const {
+    headingTitle,
+    breadcrumbs,
+    authorName,
+    authorTitle,
+    subHeadingText,
+    heroImageSrc,
+    authorImageSrc,
+  } = props;
+  return (
+    <OakThemeProvider theme={oakDefaultTheme}>
+      <OakBox $width={"100%"} $background={"bg-decorative3-main"}>
+        <OakMaxWidth>
+          <OakFlex
+            $minWidth={"100%"}
+            $flexDirection={["column-reverse", "column-reverse", "row"]}
+            $alignItems={["center", null]}
+            $minHeight={"all-spacing-20"}
+            $background={"bg-decorative3-main"}
+            $zIndex={"neutral"}
+            $justifyContent={["space-between"]}
+            data-testid="oak-header-component"
+          >
+            <OakFlex
+              $alignSelf={"flex-start"}
+              $ph={"inner-padding-m"}
+              $height={"100%"}
+              $flexDirection={"column"}
+            >
+              <OakBox
+                $display={["none", "none", "block"]}
+                $pv={"inner-padding-xl"}
+              >
+                {breadcrumbs}
```

Comment:
I think the container should be conditional since breadcrumbs is optional (or breadcrumbs should be required if that makes sense for this component)

Reply:
✅